### DISCUSSION TITLE
When I copy paste documents with tasks, checkbox renders differently between Word and Outlook

### DISCUSSION THREAD

**Discussion by aravindk100** [mvdkwast/obsidian-copy-as-html#29] [General]
I noticed that checkboxes in Word looks good but in Outlook, empty check boxes has a red cross in them. Is there a way I can customize the stylesheet to avoid this? Most of my workflow uses copy pasting in Outlook for sending meeting minutes and I have not been able to find a workaround for this. I have not observed the same issue when I export through pandoc plugin. 

Word rendering
<img width="117" alt="image" src="https://user-images.githubusercontent.com/4916255/236918500-636a3ffa-ba1c-4175-86b7-467a789a41b6.png">

Outlook rendering
<img width="103" alt="image" src="https://user-images.githubusercontent.com/4916255/236918673-71632638-0ccb-4c3f-bab5-d9458e1c93cf.png">

**Comment by mvdkwast** [comment]
Hi @aravindk100, thanks for reporting. I don't use Outlook, so I have no ready-made solution for you. Moving this to "issues" so it may get more visibility.

**Comment by mvdkwast** [comment]
Closing the discussion as there is an issue for it (#36).
